### ISSUE TITLE
Clarity on Assignee and Editor Process

### ISSUE THREAD

**Issue by andorsk** [trustoverip/TechArch#46]
Clarification on how editors reach out to assign a task.

**Comment by andorsk** [comment]
@wenjing brought up a good point during the October 13 APAC call, which was that we need to have a clarity on how editors assign tickets to the assignee.

**Comment by andorsk** [comment]
Happy to own this document if that helps.

**Comment by andorsk** [comment]
Issue with Jo getting tagged. To check up.

**Comment by andorsk** [comment]
@elisatrevino can you add Jo to the repo/org? Handle: @jospencer-460

**Comment by andorsk** [comment]
#49 has some work around this and is in DRAFT.

**Comment by andorsk** [comment]
Documentation on 
Contributors and self assignment: https://github.com/andorsk/TechArch/blob/governance/GOVERNANCE.md#activities
And Editors https://github.com/andorsk/TechArch/blob/governance/GOVERNANCE.md#activities-1 and finding the assignee

**Comment by andorsk** [comment]
@neil, that's not accurate. The process outlined in the basic flow does not require any editor and is entirely done over the GIthub UI. 

See here:
https://github.com/trustoverip/TechArch/blob/e6ea772def4706dc3c65a71b9fec238a25943d66/CONTRIBUTING.md#basic-change-flow

If this process is not sufficient, then we should discuss what would be. As it is, we've tried to simplify it as much as possible, but happy to get feedback if there's a better way you have in mind.

**Comment by neiljthomson** [comment]
@andorsk let me clarify. 

I confirmed last night in the APAC TATF call with Drummond that it was his understanding (which he announced in the NA/EU TATF call) that Contributors would require to have a local/desktop instance of Git and the TATF repo in order to make contributions (e.g. add, edit .MD docs, illustrations/images) vs the other roles in the process (Reviewers, Maintainers, Admins)

I pointed out that in reviewing Contributing.MD and watching the video, that the process for TATF did not require a "local/desktop instance" of the Repo as all of the possible actions (including PRs, and editing .md files) were possible using the GitHub Web UI and web editing. 

To which Drummonds response was, "that's great".

So the "process is sufficient", no action required.
